casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -686,6 +686,13 @@ object BlockAPI {
       case None             => none[BlockMessage]
     }
 
+  def getBlockFromStore[F[_]: Monad: BlockStore](
```

Comment:
`getBlockFromStore` and `findBlockFromStore` have same input and output args - why not have only `findBlockFromStore`? Is it performance optimisation?